File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/EngineState.scala

package org.openapitools.client.model


case class EngineState (
    /* Timestamp in Unix epoch milliseconds. */
    _timeMs: Any,
    _value: String
)
object EngineState {
    def toStringBody(var_timeMs: Object, var_value: Object) =
        s"""
        | {
        | "timeMs":$var_timeMs,"value":$var_value
        | }
        """.stripMargin
}

File: openapi-generator/dart-jaguar/lib/model/engine_state.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:openapi/model/object.dart';
part 'engine_state.jser.dart';

class EngineState {
   /* Timestamp in Unix epoch milliseconds. */
  @Alias('timeMs')
  final Object timeMs;
  
  @Alias('value')
  final String value;
  //enum valueEnum {  Running,  Off,  Idle,  };

  EngineState(
      

{
    
     this.timeMs = null,  
     this.value = null 
    }
  );

  @override
  String toString() {
    return 'EngineState[timeMs=$timeMs, value=$value, ]';
  }
}

@GenSerializer(nullableFields: true)
class EngineStateSerializer extends Serializer<EngineState> with _$EngineStateSerializer {

}
